src/main/java/frc/robot/Robot.java

```diff
@@ -170,7 +170,41 @@ public void buttonsInit() {
                     new RunCommand(() -> DriveSubsystem.drive(buttons.driverControl.getRawAxis(buttons.driveSpeedAxis),
                             buttons.driverControl.getRawAxis(buttons.driveTurnAxis)), driveSubsystem));
         }
-
+            
+    
+          
+            
+    
+        //for intake subsystem button wiring
+        //change the new methods declared to the old ones and fix this function
+        //then make sure the buttons are wired up
+        //--past ib to future ib
+
+        buttons.operatorIntakeIn
+            .whenPressed(() -> {
+                intakeSubsystem.spinForward();
+        })
+            .whenReleased(() -> {
+                intakeSubsystem.stopSpinning();
+            })
+        ;
+
+        buttons.operatorIntakeOut
+            .whenPressed(() -> {
+                intakeSubsystem.spinBackward();
+            })
+            .whenReleased(() -> {
+                intakeSubsystem.stopSpinning();
+            })
+        ;
+
+        buttons.operatorToggleIntake
+            .whenPressed(() -> {
+                intakeSubsystem.startSpinning();
+                intakeSubsystem.spinForward();
+            })
+        ;
```

Comment:
This should Jost call intakeSubsystem.toggle() and the toggle function should toggle the intake pneumatic